GitCommands/RevisionReader.cs

```diff
@@ -416,7 +418,7 @@ DateTime ParseUnixDateTime()
             // NOTE the convention is that the Subject string is duplicated at the start of the Body string
             // Therefore we read the subject twice.
             // If there are not enough characters remaining for a body, then just assign the subject string directly.
-            var (body, additionalData) = ParseCommitBody(reader, subject);
+            var body = reader.ReadToEnd().Trim();
             if (body is null)
```

Comment:
❗  Possible NRE: `ReadToEnd()` can return a `null`, hence the following line checking for `null`.

Perform the trimming inside the method.

Reply:
See above

Reply:
I changed this squashed in the latest commit.